packages/flutter/lib/src/services/clipboard.dart

```diff
@@ -32,6 +32,32 @@ class Clipboard {
   /// Used with [getData].
   static const String kTextPlain = 'text/plain';
 
+
+  /// Whether or not the clipboard is empty.
+  ///
+  /// If the user has recently started their device and the user has not yet
+  /// copied anything, this value will be true.
+  static bool get isEmpty => _isEmpty;
```

Comment:
Not sure we want 2 getters, especially since this one doesn't make it clear that the value is cached.